internal/errors/errors.go

```diff
@@ -0,0 +1,10 @@
+package errors
+
+import "errors"
+
+var ErrNotFound = errors.New("token not found")
```

Comment:
Looks good, unfortunately an Error can't be a constant. Some libraries follow the same approach.